packages/components/src/templates/next/components/internal/Siderail/SiderailLabel.tsx

```diff
@@ -0,0 +1,57 @@
+import { BiRightArrowAlt } from "react-icons/bi"
+import { tv } from "tailwind-variants"
+
+import type { Item } from "./types"
+import { useSiderailContext } from "./SiderailContext"
+
+const createSiderailLabelStyles = tv({
+  slots: {
+    container:
+      "group flex gap-1.5 px-3 py-4 transition hover:text-brand-interaction",
+    icon: "h-6 w-6 opacity-0 transition group-hover:translate-x-1 group-hover:opacity-100",
+  },
+  variants: {
+    isCurrent: {
+      true: {
+        container: "text-brand-interaction",
```

Comment:
nit: design issue but the contrast between the `text-brand-interaction` and the base text colour is very negligible, which make it difficult to tell exactly which page is the one that is the current page on the siderail

Reply:
ya told @sehyunidaaa liow she will give more think. Or just is the site theme's fault HAHA